homeassistant/components/denonavr/strings.json

```diff
@@ -4,7 +4,7 @@
     "step": {
       "user": {
         "title": "Denon AVR Network Receivers",
-        "description": "Connect to your receiver, if the IP address is not set, auto-discovery is used",
+        "description": "Connect to your receiver, if the [%key:common::config_flow::data::ip%] is not set, auto-discovery is used",
```

Comment:
Please don't mix in references inside a sentence. It's ok to have a reference as a separate sentence in a value.

Reply:
Used the references in mix with sentences, as I saw a couple of accepted and merged PRs which did the same, so thought would be of help... **Added a commit to revert** if this is supposed to be the intended way.